doc/src/sphinx/develop/contribute_docs.rst

```diff
@@ -0,0 +1,78 @@
+.. index:: How to Contribute Documentation
+
+How to Contribute Documentation
+=====================================
+
+Documentation is stored in ``doc/src/sphinx``. 
+
+Dependencies are stored in ``doc/src/sphinx/requirements.txt``.
+
+
+
+
+.. tab-set::
+
+  .. tab-item:: conda
+
+    Create a virtual environment for documentation: 
+
+    .. code:: bash
+
+      conda create -n clowder_docs python=3.7 -y
+      conda activate clowder_docs
+
+    Install doc dependencies. It's always better to run all conda commands before installing pip packages.
+
+    .. code:: bash
+
+      conda install sphinx==3.1.2 recommonmark==0.6.0 jinja2==3.0.1 m2r2==0.3.2 -y
+      pip install sphinx-rtd-theme==0.5.0 sphinx_design==0.0.13 docutils==0.16 certifi==2021.5.30 sphinx-autobuild==2021.3.14
+
+  .. tab-item:: pyenv
+
+    Create a virtual environment for documentation:
+
+    .. code:: bash
+
+      pyenv install 3.7
+      pyenv virtualenv 3.7 clowder_docs
+
+      # make virtual environemnt auto-activate
+      cd doc/src/sphinx
+      pyenv local clowder_docs
+
+    Install doc dependencies: 
+
+    .. code:: bash
+
+      pip install sphinx==3.1.2 recommonmark==0.6.0 jinja2==3.0.1 m2r2==0.3.2 sphinx-rtd-theme==0.5.0 sphinx_design==0.0.13 docutils==0.16 certifi==2021.5.30
```

Comment:
are these the same as in the requirements.txt? would it be better to use:
```
pip install -r develop/requirements.txt
```
this way we don't have to keep updating the documentation.

Reply:
Sounds great. I implemented that change. I didn't do it at first because it's not as easy for conda users, but it's easy enough and this is way more maintainable. 

Thanks!